setup.py

```diff
@@ -19,7 +21,7 @@ def get_version():
     packages=find_packages(),
     zip_safe=False,
     install_requires=[
-        'django>=1.8,<2.0',
+        'django>=1.8,<2.1',
```

Comment:
This doesn't seem like a PEP 8 or redundancy code change. Probably should be in its own commit.

Reply:
fixed